src/anbox/cmds/container_manager.cpp

```diff
@@ -55,6 +55,9 @@ anbox::cmds::ContainerManager::ContainerManager()
   flag(cli::make_flag(cli::Name{"use-rootfs-overlay"},
                       cli::Description{"Use an overlay for the Android rootfs"},
                       enable_rootfs_overlay_));
+  flag(cli::make_flag(cli::Name{"use-squashfuse"},
```

Comment:
Can we call this "--force-squashfuse" instead?

Reply:
updated